### PR TITLE
Fix for #34

### PR DESCRIPTION
**Pull request by wingedfox**

### CODE REVIEW

File: tasks/jsinspect.js [hunk 1]
```diff
@@ -64,6 +64,10 @@ module.exports = function(grunt) {
     var writableStream;
     if (typeof options.outputPath === 'string') {
       // The user wants the output to be written to a file, so pass a writable stream as an option to the reporter.
+      var path = options.outputPath.split(/[/\\]/g).slice(0, -1).join('/');
```
Comment (paazmaya) [inline tasks/jsinspect.js]: How about using 'path.dirname'?

**Comment by wingedfox** [comment]
Thanks, updated

File: tasks/jsinspect.js [hunk 2]
```diff
@@ -64,6 +65,10 @@ module.exports = function(grunt) {
     var writableStream;
     if (typeof options.outputPath === 'string') {
       // The user wants the output to be written to a file, so pass a writable stream as an option to the reporter.
```
Comment (stefanjudis) [inline tasks/jsinspect.js]: This comment looks a bit weird now, or is it just me?

File: tasks/jsinspect.js [hunk 3]
```diff
@@ -92,13 +97,17 @@ module.exports = function(grunt) {
       writableStream.on('finish', function() {
         done(taskSucceeded);
       });
+      writableStream.on('error', function() {
+        throw new Error('Can not write to ' + options.outputPath);
```
Comment (stefanjudis) [inline tasks/jsinspect.js]: Are there options that there can be a different error than

```
Fatal error: ENOENT, open 'target/reports/jsinspect/output.xml'
```

If so, we might want to provide some more information here?
Would have to doublecheck, though. But the original error provides at leas `ENOENT`, which gives a hint.

File: tasks/jsinspect.js (hunk 2 above)
Comment (wingedfox) [inline tasks/jsinspect.js]: Probably it should be changed somehow, but it is understandable

File: tasks/jsinspect.js (hunk 3 above)
Comment (wingedfox) [inline tasks/jsinspect.js]: As for me, it is not a big hint. Possible cases: wrong access rights, folder with the same name exists, intermediate folders were not created, some internal stream errors.
At best, total error handling should be provided, but this is a part of the separate request.

Comment (stefanjudis) [inline tasks/jsinspect.js]: Ja agree - but we could still pass the original error msg with the custom one, to provide a better experience. :)

**Comment by wingedfox** [comment]
PR updated

**Comment by stefanjudis** [comment]
Not exactly what I had in mind, but let's do it. :)

Thanks a lot for taking the time. :bowtie:

**Comment by wingedfox** [comment]
As long as stream reports EACCESS as only the warning, there's no good way to process errors.
```
Warning: EACCES, permission denied 'target/test' Used --force, continuing.
```

**Pull request merged**